charts/lakefs/templates/secret.yaml

```diff
@@ -14,3 +14,58 @@ data:
   auth_encrypt_secret_key: {{ .Values.secrets.authEncryptSecretKey | default "" | b64enc }}
 {{- end }}
 {{- end }}
+---
+
+# certificates to use with SAML SSO against server such as AD FS
+
+{{- if (.Values.enterprise).enabled }}
+{{- if (((.Values.enterprise).auth).saml).enabled }}
+{{- if (((.Values.enterprise).auth).saml).createCertificateSecret }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: saml-certificates
+data:
+  rsa_saml_public.pem: '{{ .Values.enterprise.auth.saml.certificate.samlRsaPublicCert | b64enc }}'
+  rsa_saml_private.key: '{{ .Values.enterprise.auth.saml.certificate.samlRsaPrivateKey | b64enc }}'
+{{- end }}
+{{- end }}
+{{- end }}
+---
+# LDAP bind password secret, used for LDAP authentication
+{{- if (.Values.enterprise).enabled }}
+{{- if (((.Values.enterprise).auth).ldap).enabled }}
+{{- if (((.Values.enterprise).auth).ldap).bindPassword }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ldap-secret
```

Comment:
No need in additional secret just add that to the lakefs-secret

Reply:
Let's continue discussion in the OIDC duplicated [comment](https://github.com/treeverse/charts/pull/343#discussion_r2188335104)